ledger-tool/src/main.rs

```diff
@@ -2214,21 +2229,15 @@ fn main() {
         .map(BlockstoreRecoveryMode::from);
     let force_update_to_open = matches.is_present("force_update_to_open");
     let verbose_level = matches.occurrences_of("verbose");
-
     // TODO: the following shred_storage_type inference must be updated once the
     // rocksdb options can be constructed via load_options_file() as the
     // temporary use of DEFAULT_LEDGER_TOOL_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES
     // could affect the persisted rocksdb options file.
-    let shred_storage_type = match ShredStorageType::from_ledger_path(
+    let shred_storage_type = get_shred_storage_type(
         &ledger_path,
         DEFAULT_LEDGER_TOOL_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES,
```

Comment:
Do you think this comment should go with the helper? Also, thoughts on moving the constant inside the helper ? Less places to update once/if we address what the TODO is saying